### PR TITLE
OrphanedVirtualMachineInstances edits

### PR DESCRIPTION
**Pull request by apinnick**
Signed-off-by: Avital Pinnick <apinnick@redhat.com>

### CODE REVIEW

**Comment by apinnick** [comment]
@sradco Ready for review

File: docs/runbooks/OrphanedVirtualMachineInstances.md [hunk 1]
```diff
@@ -1,77 +1,65 @@
...
-### Unhealthy virt-handler DaemonSet
-Check the DaemonSets status to see what the issues are with deploying the pods.
-
-`kubectl describe daemonset virt-handler --all-namespaces`
-
-
-Another way to check the status is to get the object and read through its status object.
-
-`kubectl get daemonset virt-handler --all-namespaces -o yaml | jq .status`
+If the `virt-handler` daemon set is not healthy:
```
Comment (sradco) [inline docs/runbooks/OrphanedVirtualMachineInstances.md:35]: @apinnick I think it should be DaemonSet

Comment (sradco) [reply docs/runbooks/OrphanedVirtualMachineInstances.md:35]: also in the rest of the places in this runbooks

Comment (apinnick) [reply docs/runbooks/OrphanedVirtualMachineInstances.md:35]: No, it should be "daemon set".

**Pull request merged**